vspace/src/main/webapp/WEB-INF/views/staff/modules/module.html

```diff
@@ -20,6 +20,29 @@
     });
 } 
 
+function deleteModule(moduleId){
+	 $.ajax({
+		url: "[(@{'/staff/module/'+${module.id}+'/delete'})]"+'?'+[[${_csrf.parameterName}]]+'='+[[${_csrf.token}]],
+	        type: 'DELETE',
+	        cache       : false,
+	        contentType : false,
+	        success: function(data) {
+	        	var alert = $('<div class="alert alert-dismissible fade show" role="alert"><p>This module has been successfully deleted.</p><button type="button" class="close" data-dismiss="alert" aria-label="Close"><span aria-hidden="true">&times;</span></button></div>');
+	            $('.error').append(alert);
+	            var startSequence = document.getElementById("startSequence");
+	            var sequences = document.getElementById("sequences");
+	            var slides = document.getElementById("slides");
+	            startSequence.remove();
+	            sequences.remove();
+	            slides.remove();
```

Comment:
does this mean the user ends up on a module page for a module that doesn't exist anymore? That doesn't make sense. They should be redirected to the module list page.

Reply:
Have redirected now